web_exportall/AUTHORS

```diff
@@ -0,0 +1 @@
+Craig Gowing <craig.gowing@credativ.co.uk>
```

Comment:
Authors file is not used on OCA modules. Please add it to the README file and the manifest file.